code_review_guidline/main.dart

```diff
@@ -31,11 +31,16 @@ class GuessingGame {
     player.GussNumber = guassNumber;
     if (player.GussNumber < numberToBeGuessed) {
       player.isLower();
+      player.isWin = false;
     } else if (player.GussNumber > numberToBeGuessed) {
       player.isGreater();
-    } else {
+      player.isWin = false;
+    } else if (player.GussNumber == numberToBeGuessed) {
```

Comment:
your code looks good! consider the following
use lowerCamelCase for  your GussNumber declaration
and your code has some type errors

Reply:
solved for camelCase

Reply:
the class where in the correct format, just consider for class properties

Reply:
solved for typo error